Source/DafnyCore/Dafny.atg

```diff
@@ -3202,6 +3203,52 @@ Forall = "forall".
 Exists = "exists".
 QSep = "::".
 
+Expression<out Expression e, bool allowLemma, bool allowLambda, bool allowBitwiseOps = true>
+=
+  NotDecreasesToExpression<out e, allowLemma, allowLambda, allowBitwiseOps>
+  .
```

Comment:
I have no idea how to improve on that, but making this the top level seems like a big change.

Reply:
Yeah, and now that you mention it this change isn't necessary anymore. I made it to help with switching certain things back and forth during debugging, but in the final version I don't think it's necessary. I'll try to remove it.